dev-erlang/epam/epam-1.0.9.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
-inherit rebar user
+inherit rebar
```

Comment:
I’d suggest revbumping for these to try get simpler bug reports (you know everybody’s on the same, new GLEP 81 version).